manifests/params.pp

```diff
@@ -4,30 +4,30 @@
   $ssl_dir = '/etc/puppetlabs/puppet/ssl'
 
   # Set CA file
-  if defined('$::rh_certificate_repo_ca_file') {
-    $rh_certificate_repo_ca_file = $::rh_certificate_repo_ca_file
+  if defined('$facts[rh_certificate_repo_ca_file]') {
```

Comment:
This is tricky: I think these are actually global variables defined in the ENC. https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/plugin/remote_execution/ssh_user/params.pp has more examples of that.

Reply:
I think those are just facts?

https://github.com/theforeman/puppet-foreman_scap_client/blob/095807f99366357c58acc0a2dd11111b34015570/lib/facter/rh_certificates.rb#L22-L38

Reply:
Ah right. In that case you can ignore my comment. It is tempting to make it a structured fact, but that's outside of the scope now :)

Reply:
Hush, go, dig an own rabbit hole :stuck_out_tongue: